pages/common/gnucash-cli.md

```diff
@@ -0,0 +1,12 @@
+# gnucash-cli
+
+> A command-line version of GnuCash.
+> More information: <https://manned.org/gnucash-cli>.
+
+- Get online quotes for currencies and stocks in the file:
+
+`gnucash-cli --quotes get {{path/to/file.gnucash}}`
+
+- Generate a report:
```

Comment:
What does this command do? Right now, it sounds like it's creating a crash or bug report.

Reply:
Ahh, sorry about that. It generates financial reports from the specified data file (`example.gnucash`), such as a "Profit & Loss", "Balance Sheet", etc and logs the output to the console.

Reply:
Ah, okay. I thought it could be any arbitrary name. Then how about
```diff
- - Generate a financial report:
+ - Generate a financial report of a specific type, specified by `--name`:
```

Reply:
This is one I was a bit confused with how to word, as I felt the name of the argument was terribly ambiguous.
I think this is much better.